rllib/agents/dqn/dqn.py

```diff
@@ -47,7 +47,14 @@
         "initial_epsilon": 1.0,
         "final_epsilon": 0.02,
         "epsilon_timesteps": 10000,  # Timesteps over which to anneal epsilon.
+
+        # For soft_q, use:
+        # "exploration" = {
+        #   "type": "SoftQ"
```

Comment:
What's the right name for this? StochasticSampling vs SampleFromDistribution vs ???

Reply:
SoftQ is a sub-class of StochasticSampling (adding the temp as convenience).